tests/Fake_data_generator_test.py

```diff
@@ -0,0 +1,62 @@
+'''
+This scripts aims at testing Tritium specific processors.
+Author: C. Claessens
+Date: Apr 6 2020
+'''
+
+import unittest
+
+from morpho.utilities import morphologging
+logger = morphologging.getLogger(__name__)
+
+import matplotlib.pyplot as plt
+
+class FakeDataGenerationTest(unittest.TestCase):
+
+    def test_data_generation(self):
+        from mermithid.processors.TritiumSpectrum.FakeDataGenerator import FakeDataGenerator
+        from mermithid.misc.Constants import seconds_per_year, tritium_endpoint
+
+        specGen_config = {
+            "apply_efficiency": False,
+            "efficiency_path": "/host/input_data/combined_energy_corrected_eff_at_quad_trap_frequencies.json",
```

Comment:
For testing this processor during CI, it might be worth implement a fake detection efficiency file and add it to the tests folder